assets/default/css/custom.css

```diff
@@ -6,5 +6,6 @@
    ========================================================================== */
 
 .footer-including-page-number {
```

Comment:
*Never* add anything to the custom.css file. Please move this to the app scss file.